include/suiryoku/locomotion/control/node/control_node.hpp

```diff
@@ -37,22 +37,24 @@ class ControlNode
 {
 public:
   using RunLocomotion = suiryoku_interfaces::msg::RunLocomotion;
-  using Status = suiryoku_interfaces::msg::Status;
+  using Bool = std_msgs::msg::Bool;
```

Comment:
looks like that causes an error when i tried to build the package since its inside of the class scope, here's the message that i got
```
/home/faza/ws_ros/src/suiryoku/include/suiryoku/locomotion/control/node/control_node.hpp:40:24: error: using-declaration for non-member at class scope
   40 |   using std_msgs::msg::Bool;
      |                        ^~~~
```

Reply:
ooh right, not realized that it doesn't work inside class scope.